src/_locales/ru/messages.json

```diff
@@ -60,76 +60,76 @@
-    "html_options_suspend_screen_capturing_screen_only": { "message": "Захватывать только верх экрана" },
+    "html_options_suspend_screen_capturing_screen_only": { "message": "Захватывать только верх страницы" },
     "html_options_suspend_screen_capturing_entire_page": { "message": "Захватывать всю страницу" },
-    "html_options_suspend_screen_capturing_tooltip_line1a": {"message": "На заблокированной вкладке расширение может отображать предварительный просмотр изображения вкладки"},
-    "html_options_suspend_screen_capturing_tooltip_line1b": {"message": " как раз перед тем, как она была приостановлена." },
-    "html_options_suspend_screen_capturing_tooltip_line2a": {"message": "Функция захвата экрана является экспериментальной и может привести к значительным нагрузке на процессор"},
-    "html_options_suspend_screen_capturing_tooltip_line2b": {"message": "а также повышенное использование памяти." },
-    "html_options_suspend_screen_capturing_tooltip_line3a": { "message": "Если вы заметили странное поведение, например долгая приоставновка вкладки, или Chrome" },
-    "html_options_suspend_screen_capturing_tooltip_line3b": { "message": "неожиданно падает, попробуйте отключить эту функцию." },
-    "html_options_suspend_force_screen_capture": { "message": "Включить режим высококачественного захвата экрана" },
-    "html_options_suspend_force_screen_capture_tooltip_line1": { "message": "Когда отключено, снимки экрана имеют следующие ограничения для обеспечения производительности:" },
-    "html_options_suspend_force_screen_capture_tooltip_line2": {"message": "- Низкое качество захвата экрана"},
-    "html_options_suspend_force_screen_capture_tooltip_line4": {"message": "- Не будет отображаться экран, если для генерации изображения требуется более 1 минуты"},
-    "html_options_suspend_force_screen_capture_tooltip_line5": {"message": "- Максимальная высота захвата экрана ограничена 5000px"},
-    "html_options_suspend_force_screen_capture_tooltip_line6": {"message": "Включение режима высококачественного захвата экрана может значительно увеличить загрузку процессора"},
-    "html_options_suspend_force_screen_capture_tooltip_line7": {"message": " и использование памяти." },
+    "html_options_suspend_screen_capturing_tooltip_line1a": {"message": "На приостановленной вкладке расширение будет отображать скриншот страницы"},
+    "html_options_suspend_screen_capturing_tooltip_line1b": {"message": "до ее приостановки." },
+    "html_options_suspend_screen_capturing_tooltip_line2a": {"message": "Скриншот страницы является экспериментальной функцией и может привести к значительной"},
+    "html_options_suspend_screen_capturing_tooltip_line2b": {"message": "нагрузке на процессор, а также к повышенному потреблению памяти." },
+    "html_options_suspend_screen_capturing_tooltip_line3a": { "message": "Если Вы заметили странное поведение, например, долгую приостановку вкладки или" },
```

Comment:
`вы` (instead of `Вы`) is correct: http://new.gramota.ru/spravka/letters/51-rubric-88

Also, "длительная приостановка" is better, I think.

Reply:
We r appealing to a **single person** - user (even if we r don't know that person but just as a form of some extra "respect"), so I think capital "Вы" is good here, isn't it nice if program/extension being that polite with u?

Reply:
> Употребление местоимения вы вместо ты при обращении к одному лицу само по себе уже представляет проявление уважительного отношения к этому лицу. Окончательное решение о написании Вы с прописной (для подчеркивания этого уважительного отношения) принимает автор текста.

So… "вы" is OK, at least for me. I understood "Вы" like excessive flattery. But it's decision of author, OK. "Вы" is acceptable in place like this, as I understood.

Reply:
We "excessively flatter" our "beloved" user so that user donates more to the dev of this Great extension? Why not to be that nice? It's not that formal extension with all those funny suspendy faces 😁